app/js/translations.js

```diff
@@ -10,14 +10,18 @@ module.exports = {
 
     i18n.use(fsBackend).init({
       backend: {
-        loadPath: path.join(__dirname, '../../locales/{{lng}}.json')
+        loadPath: path.join(__dirname, '../../locales/__lng__.json')
       },
 
       // Load translation files synchronously: https://www.i18next.com/overview/configuration-options#initimmediate
       initImmediate: false,
 
-      // We use the i18next v1 JSON format: https://www.i18next.com/misc/json-format#i-18-next-json-v1
-      compatibilityJSON: 'v1',
+      // We use the legacy v1 JSON format, so configure interpolator to use
+      // underscores instead of curly braces
+      interpolation: {
+        prefix: '__',
+        suffix: '__'
```

Comment:
Note that this will break translations from locales with a `-` next to `__`. IIRC there is yet another option for changing the escape character or something like that.

`pre __appName__- middle __otherField__ post` is omitting the `middle` part.

Reply:
Thanks, it was on my list to check exactly what the interpolation settings were for the old syntax. The docs point here: https://github.com/i18next/i18next/blob/5fb4dbcb7ccdcfc78192e69ef3291d2ace6814b4/test/backward/compatibility/v1.js#L4 for how they convert their old tests.